docs/contributor-guide/developer-guide/releasing.md

```diff
@@ -2,23 +2,42 @@
 
 See [Upgrading to a New Release](it-manual/sre-playbook/upgrading-to-a-new-release.md) for an overview of CiviForm release practices.
 
-Releases are created once a month by the on-call engineer, and more often if requested by a civic entity (e.g. for an urgent bugfix).
+Releases are created weekly by the on-call engineer, usually on Wednesday, or if requested by a civic entity (e.g. for an urgent bugfix).
 
 ## Creating a new release
 
 ### 1. Find the most recent commit that is passing CI for all staging environments:
 
-* [Seattle staging](https://staging.seattle.civiform.com/)
+Find the commit SNAPSHOT id from one of the following, and extract the SHA from it. The SHA is the middle item in the SNAPSHOT-sha-hash ID
+
+#### From Deployment-bot
+
+Check the #deployment-bot Slack channel for the latest successful SNAPSHOT for Azure and AWS.  This offers a history of previous auto pushes too.
+
+#### From Staging servers
+
+On the running servers inspect the DOM tree for the <head><meta> item civiform-build-tag. This will only show the current recent for each.
```

Comment:
```suggestion
On the running servers inspect the DOM tree for the <head><meta> item civiform-build-tag. This will only show the current SNAPSHOT for each though.
```